maxim/tests/test_connection_retry_logic.py

```
@@ -0,0 +1,526 @@
+from urllib3.exceptions import ProtocolError, MaxRetryError, PoolError
+from requests.exceptions import (
+    ConnectionError,
+    RequestException,
+    ConnectTimeout,
+    ReadTimeout,
+    HTTPError,
+)
+from http.client import RemoteDisconnected
+
+# Add the maxim package to the path
+import sys
```

Pylint (3.3.7)

[convention] standard import "sys" should be placed before third party imports "urllib3.exceptions.ProtocolError", "requests.exceptions.ConnectionError"

(C0411)